react-components/src/components/Architecture/RevealButtons.tsx

```diff
@@ -10,20 +10,31 @@ import { SetFlexibleControlsTypeCommand } from '../../architecture/base/concrete
 import { SetAxisVisibleCommand } from '../../architecture/concrete/axis/SetAxisVisibleCommand';
```

Comment:
Can not see the purpose of this way of doing it anymore. The way it was done before your changes is a lot cleaner.